### DISCUSSION TITLE
[Question] Automatically enable Pro Trial?

### DISCUSSION THREAD

**Discussion by StefanWin** [craftcms/cms#10604] [Q&A] [answered]
### Description

(Sorry for marking this as a bug but there is no way to change labels)

Is there a way to automatically (i.e. w/o the web UI) enable the pro trial? We're using craft w/ graphql as a headless CMS for a static-site-generator and we're building a local docker setup for testing and development.

Manually setting the `system.edition` to `pro` in `project.yaml` does not work. Editing the database entry in `projectconfig` table also doesn't work.  
With docker volumes, this is generally a minor inconvenience since it only has to be enabled manually once but being able to automatically enable the graphql endpoint would allow for automatic integration tests within a CI (would probably be smarter to extract the schema and run a mock server).

### Additional info

- Craft version: 3.7.33
- PHP version: irrelevant
- Database driver & version: irrelevant
- Plugins & versions: irrelevant

**Comment by brandonkelly** [comment, marked as answer]
> Manually setting the `system.edition` to `pro` in `project.yaml` does not work.

It does, but you need to run the following command after making that change:

```sh
php craft project-config/apply
```

**Reply by StefanWin** [reply]
I was actually running `project-config/apply` since we're seeding a bunch of custom fields but I must have made a mistake along the way. Sorry for bothering you and thank you for the reply. Works perfectly now.
